web/src/app/environments/view/environments-view.component.html

```diff
@@ -130,8 +130,11 @@ <h3 matLine>{{ environment.latestPing.url }}</h3>
           <span class="mat-caption text-right" flex="30" layout-align="row" *ngIf="!environment.latestPing.isValid">
             <p>No pings</p>
           </span>
+          <span class="mat-caption text-right" flex="30" layout-align="row" *ngIf="environment.latestPing.isValid">
+            <p>{{ monitor.pings.valid }} valid pings</p>
+            <p>{{ monitor.pings.invalid }} invalid pings</p>
```

Comment:
On this page there is no `monitor` variable (double check in the network tab), but as it is at the environment level, the variable used here is `environment`, so the `valid` and `invalid` would be `environment.pings.valid` and `environment.pings.valid`. I can explain more on our next pairing session 🤓